runtime/controller_test.go

```diff
@@ -1218,12 +1217,12 @@ measures:
 	testruntime.RequireResource(t, rt, id, metricsRes)
 }
 
-func newSource(name, path, localFileHash string) (*runtimev1.Model, *runtimev1.Resource) {
+func newSource(t *testing.T, rt *runtime.Runtime, id string, name, path, localFileHash string) (*runtimev1.Model, *runtimev1.Resource) {
 	source := &runtimev1.Model{
 		Spec: &runtimev1.ModelSpec{
 			InputConnector:   "local_file",
 			OutputConnector:  "duckdb",
-			InputProperties:  must(structpb.NewStruct(map[string]any{"path": path, "local_files_hash": localFileHash})),
+			InputProperties:  must(structpb.NewStruct(map[string]any{"path": "data/foo.csv", "local_files_hash": localFileHash})),
```

Comment:
What is the point of any of the changes in this file?

Reply:
It was initially bawking at a previous change when some fields were not optional. I can drop the changes